### PR TITLE
[SPARKTA-209][Driver] Hdfs utils changed to allow remote connections.

### PR DESCRIPTION
**Pull request by sgomezg**
#### Description
Hdfs utils changed to allow remote connections. SPARKTA-209
Doc updated: https://stratio.atlassian.net/wiki/display/SPARKTA0x9/6.+Configuration#id-6.Configuration-5.6.Clusterconfiguration

#### Reviewer
@anistal, @danielcsant 

#### Test
All OK

### CODE REVIEW

File: serving-api/src/main/resources/reference.conf [hunk 1]
```diff
@@ -36,8 +36,8 @@ sparkta {
 
   hdfs {
     hadoopUserName = stratio
-    hadoopConfDir = /home/ubuntu
     hdfsMaster = localhost
+    hdfsPort = 8020
```
Comment (danielcsant) [inline serving-api/src/main/resources/reference.conf]: Please make this changes also in `dist/src/main/resources/application.conf` file

Comment (sgomezg) [inline serving-api/src/main/resources/reference.conf]: Added, thanks

**Comment by anistal** [comment]
+1

**Comment by danielcsant** [comment]
+1

**Pull request merged**